app/src/main/java/com/teamagam/gimelgimel/app/utils/PreferenceUtil.java

```diff
@@ -23,6 +27,17 @@ public PreferenceUtil(Resources res, SharedPreferences preferences) {
         mPref = preferences;
     }
 
+    /*
+     * some static util functions
```

Comment:
Either write a meaningful comment (a comment that explains something the code fails to pass) or don't write one at all !fix